frontend/pages/api/pdf/[pathName].ts

```diff
@@ -0,0 +1,17 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import fetch from 'node-fetch';
+
+export default async function passThroughPdf(req: NextApiRequest, res: NextApiResponse) {
```

Comment:
This code still looks a bit dangerous to me? Both because fetching large files crashes stuff and the previously discussed risks of passing through arbitrary files. Hopefully it's fine. Either way, we should handle fetch errors so we only send the user a PDF if response.status === 200.